### ISSUE TITLE
Is it possible to create an entity using a regular expression?

### ISSUE THREAD

**Issue by Luccasoli** [facebook/draft-js#2445]
a question, is it possible to create an entity using a regular expression? (the entity would start with a # and end when typing a non-alphanumeric character)

**Comment by aidanoconnor1** [comment]
did you find an answer to this questions?

**Comment by Luccasoli** [comment]
No :sob:

**Comment by rahul1995** [comment]
Do you mean the decorator? Like a hashtag decorator to make it blue? Read this: https://draftjs.org/docs/advanced-topics-decorators#compositedecorator

**Comment by Luccasoli** [comment]
@rahul1995 No, I really want entities, I needed to save information associated with a string preceded by a "#"

**Comment by hellodoli** [comment]
@Luccasoli same question, I want entity with hashtag but I don't know how.
